app/models/ingest_job.rb

```diff
@@ -509,6 +511,13 @@ def create_differential_expression_results
     de_result.save
   end
 
+  # set flags to denote when a cluster has image data
+  def set_has_image_data
+    Rails.logger.info "Setting image_pipeline flags in #{study.accession} for cluster: #{study_file.name}"
+    cluster_group = ClusterGroup.find_by(study_id: study.id, study_file_id: study_file.id)
+    cluster_group.update(has_image_data: true) if cluster_group.present?
```

Comment:
Blocker: Clearer intent, less ambiguous:

```suggestion
  def set_has_image_cache
    Rails.logger.info "Setting image_pipeline flags in #{study.accession} for cluster: #{study_file.name}"
    cluster_group = ClusterGroup.find_by(study_id: study.id, study_file_id: study_file.id)
    cluster_group.update(has_image_cache: true) if cluster_group.present?
```

Reply:
[69c0769](https://github.com/broadinstitute/single_cell_portal_core/pull/1663/commits/69c0769a4cb881b9c814c8e159f538b10e4298ba)